src/bitmessageqt/settings.py

```diff
@@ -1,16 +1,24 @@
 # -*- coding: utf-8 -*-
+# pylint: disable=too-many-instance-attributes,too-many-locals,too-many-statements,attribute-defined-outside-init
+"""
+Form implementation generated from reading ui file 'settings.ui'
 
-# Form implementation generated from reading ui file 'settings.ui'
-#
-# Created: Thu Dec 25 23:21:20 2014
-#      by: PyQt4 UI code generator 4.10.3
-#
-# WARNING! All changes made in this file will be lost!
+Created: Thu Dec 25 23:21:20 2014
+     by: PyQt4 UI code generator 4.10.3
+
+WARNING! All changes made in this file will be lost!
+"""
```

Comment:
It's really going to be a funny documentation (`:

Reply:
As discussed elsewhere, our process involves manually editing this file which was auto-generated once, five years ago. This comment will therefore be removed since it is misleading. If you want to suggest re-generating this file as part of the build process then please either open an issue or PR.